### ISSUE TITLE
does mi mix2s also suitable for you process?

### ISSUE THREAD

**Issue by birdnofoots** [PhucHauDeveloper/Kali-nethunter-pro-for-OnePlus-6-6T#1]
i follow the officiall process. 
fastboot flash userdata xxxxx.img
fastboot flash boot boot-polaris.img

when finished. build in wifi cant work. mibile data cant switch on. and the vioce volumn almost zero.

**Comment by birdnofoots** [comment]
and if i want to flash back to android os . such as lineage . i need do edl flash first?

**Comment by PhucHauDeveloper** [comment]
Answering your first question, I own a Mi Mix 2S, but unfortunately, the replacement screen I recently installed isn’t compatible with either Windows or Linux, as it doesn’t work properly, so I can’t test this for you. However, I believe there are a few ways to address your issue:
-You should try both the official build from kali.org and the one by Shubhamvis98 to see which one works better for you.
-If you’ve successfully flashed it but WiFi and other features aren’t working, even though this is a known issue, I recommend using EDL mode to flash back to the stock ROM before installing Kali. Some AOSP ROMs may have conflicting custom packages that cause issues with Linux.
For your second question, you **SHOULD** use **EDL mode** when switching back to Android, even between Android versions. I strongly advise flashing the stock ROM via EDL before installing any other OS. This is commonly emphasized by custom ROM developers in their guides.

**Comment by birdnofoots** [comment]
thank for you quick reply. i alraldy try the Shubhamvis98's 20240813 img. because the latest one no polaris boot img.  unfortunately, 
cant reboot normally. so i just use back the official imgs.

**Comment by PhucHauDeveloper** [comment]
Yes you can try old boot.img and new rootf to try, you may need to try many images to find the best one, finally if it is too difficult you can use nethunter on android it is still very good if you need hid you can use [USB Gadget Tool](https://github.com/tejado/android-usb-gadget) to enable HID on Android, good luck.
